crypto/bio/bss_file.c

```diff
@@ -33,8 +33,6 @@
 # include "bio_lcl.h"
 # include <openssl/err.h>
 
-# if !defined(OPENSSL_NO_STDIO)
-
```

Comment:
The code below uses stdio capabilities so removing this guard seems odd and liable to break things.

Reply:
I reverted this change.